packages/web/src/pages/artist-coins-launchpad-page/constants.ts

```diff
@@ -8,3 +8,9 @@ export enum Phase {
   REVIEW,
   BUY_COIN
 }
+
+export const LAUNCHPAD_COIN_DESCRIPTION = (
+  artistHandle: string,
+  coinSymbol: string
+) => `
+${coinSymbol} is associated with artist @${artistHandle} on Audius. It aims to attract communities in need of entertainment and connection, embodying the playful and creative spirit of meme culture, and promoting interaction between fans and collectors.`
```

Comment:
is this final copy?

Reply:
we may pull this in on the indexer side and show it later on. ig just double check the copy in the channel